idd/Energy+.idd.in

```diff
@@ -73923,6 +73923,321 @@ HeatPump:PlantLoop:EIR:Heating,
+  N1 , \field Nominal Heating Capacity
+       \autosizable
+       \minimum> 0
+       \units W
+       \note Nominal Heating Capacity in [W] (autosizeable)
+  N2 , \field Design Flow Rate
+       \autosizable
+       \minimum> 0
+       \units m3/s
+       \note Design Flow Rate in m3/s (autosizeable)
+  N3 , \field Design Supply Temperature
+       \default 60
+       \units C
+       \note Design Supply Temperature in [degree C]
+  N4 , \field Design Temperature Lift
+       \autosizable
+       \default 11.1
+       \units deltaC
+       \note Design Temperature Lift in [degree C]
+  N5 , \field Sizing Factor
+       \minimum 1.0
+       \default 1.0
+       \note Sizing Factor for equipment sizing
+  A8 , \field Flow Mode
+       \required-field
+       \type choice
+       \key NotModulated
+       \key ConstantFlow
+       \key LeavingSetpointModulated
+       \default NotModulated
+       \note Flow Mode for the water side of the fuel-fired absorption heat pump
+  A9 , \field Outdoor Air Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key DryBulb
+       \key WetBulb
+       \default DryBulb
+       \note Outdoor air temperature curve input variable;
+       \note The options are Outdoor Air Dry Bulb or Wet Bulb temperature for curves
+  A10, \field Water Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key EnteringCondenser
+       \key LeavingCondenser
+       \default EnteringCondenser
+       \note Water Temperature curve input variable - Condenser Entering or Leaving Water Temperature for curves
+  A11, \field Normalized Capacity Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BivariateFunctions
+       \note: CAPFT - Normalized Capacity Function of Temperature Curve name,
+       \note which is a biquadratic curve or a lookup table.
+  A12, \field Fuel Energy Input Ratio Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BivariateFunctions
+       \note EIRFT - Fuel Energy Input Ratio Function of Temperature Curve Name,
+       \note which is a biquadratic curve or a lookup table.
+  A13, \field Fuel Energy Input Ratio Function of PLR Curve Name
+       \required-field
+       \type object-list
+       \object-list UnivariateFunctions
+       \note EIRFPLR - Fuel Energy Input Ratio Function of Part Load Ratio(PLR) Curve Name,
+       \note which is a cubic curve or a lookup table.
+  N6 , \field Minimum Part Load Ratio
+       \minimum> 0.0
+       \maximum 1.0
+       \default 0.1
+       \note Minimum Part Load Ratio (PLR) in between 0 and 1
+  N7 , \field Maximum Part Load Ratio
+       \minimum> 0.0
+       \maximum 1.0
+       \default 1.0
+       \note Maximum Part Load Ratio (PLR) in between 0 and 1
+  A14, \field Fuel Energy Input Ratio Defrost Adjustment Curve Name
```

Comment:
This field should come after Defrost Control Type, and probably after the timed defrost inputs.

Reply:
The related defrost fields are now grouped after Defrost Control Type which was the main goal here. I could quibble about the final order of the fields, but I'll mark these conversations resolved.

Reply:
There are indeed a trail of comments on the IDD and also changes according to the comments along the way. Now I think all the IDD questions or comments have been addressed/answered.